mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulPlayScreen.kt

```diff
@@ -159,27 +169,30 @@ private fun info(match: Match): Flow<MatchInfo> {
 }
 
 /**
- * Implementation of the PlayscreenState
+ * Implementation of the PlayScreenState
  * @param onNewGameClick callback function for the new game button
  * @param user authenticated user
- * @param facade [ChessFacade] to fetch the matches
+ * @param chessFacade [ChessFacade] to fetch the matches
  * @param scope for coroutines
  */
 private class PlayScreenStateImpl(
-    onNewGameClickAction: State<() -> Unit>,
+    override val user: AuthenticatedUser,
```

Comment:
This will probably end up being

```suggestion
    private val user: AuthenticatedUser,
```

instead.

Reply:
[f26de1b](https://github.com/epfl-SDP/android/pull/226/commits/f26de1b2a5422ea1e8d648d72f665470fe948763)